lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -134,20 +134,31 @@ def self.get_or_create_app(params)
         api_token = params[:api_token]
         owner_name = params[:owner_name]
         app_name = params[:app_name]
+        app_display_name = params[:app_display_name]
+        app_os = params[:app_os]
+        app_platform = params[:app_platform]
 
         platforms = {
           "Android" => ['Java', 'React-Native', 'Xamarin'],
           "iOS" => ['Objective-C-Swift', 'React-Native', 'Xamarin']
         }
 
+        
         if Helper::AppcenterHelper.get_app(api_token, owner_name, app_name)
           true
         else
```

Comment:
nit since the if conditional above returns, we could reduce nesting and remove the else clause if we wanted

Reply:
this `else` with `UI.error("Lane aborted")` message is for case when user didn't accept `UI.confirm("App with name #{app_name} not found, create one?")`